### ISSUE TITLE
Correction to the FAQ

### ISSUE THREAD

**Issue by cdman** [pallets/jinja#553]
Here: http://jinja.pocoo.org/docs/dev/faq/#my-tracebacks-look-weird-what-s-happening

The correct source code for the latest google app engine devserver is:

````python
import os

if os.environ.get('SERVER_SOFTWARE', '').startswith('Dev'):
    from google.appengine.tools.devappserver2.python import sandbox
    sandbox._WHITE_LIST_C_MODULES += ['_ctypes', 'gestalt']
````

**Comment by jeffwidman** [comment]
Thanks! 

Would you mind submitting your patch as a PR?

**Comment by cdman** [comment]
Done: #567

**Comment by jeffwidman** [comment]
Thanks! Fix landed in e2eab553f42d6ad388740b92856e7cd4d8e0f719
